client/app/services/service-details/service-details.html

```diff
@@ -9,10 +9,10 @@
   </span>
   <div class="view-selector-bar">
     <div class="form-group toolbar-pf-view-selector">
-      <button class="btn btn-link usage-graphs-button" ng-click="vm.viewSelected('graphsView')"
-              ng-class="{active: vm.viewType === 'graphsView'}">
-        <i class="fa fa-tachometer"></i>
-      </button>
+      <!--<button class="btn btn-link usage-graphs-button" ng-click="vm.viewSelected('graphsView')"-->
+              <!--ng-class="{active: vm.viewType === 'graphsView'}">-->
+        <!--<i class="fa fa-tachometer"></i>-->
+      <!--</button>-->
```

Comment:
Why comment out VS. remove?  If that feature is not something PM wants, shouldn't we pull all the codes?